devices/fields/fsmc/sram.yaml

```diff
@@ -32,6 +35,7 @@ BCR?:
   FACCEN:
     Disabled: [0, Corresponding NOR Flash memory access is disabled]
     Enabled: [1, Corresponding NOR Flash memory access is enabled]
+  # TODO: fix
```

Comment:
Why the todo?

Reply:
There is also can be Bits32 in MWID.
Need to investigate more. In another PR maybe.